src/Menu/PerformanceRenderer.cpp

```
@@ -75,8 +83,93 @@ static void DrawProfileButtonRow(const Feature::PerfProfile (&profiles)[3], cons
+	drawList->PopClipRect();
+
+	if (hovered)
+		ImGui::SetMouseCursor(ImGuiMouseCursor_Hand);
+	return pressed;
+}
+
+static void DrawGlobalProfileChooser(const Feature::PerfProfile (&profiles)[3], const char* const (&labels)[3],
+	const char* const (&descriptions)[3], int activeIdx)
+{
+	const auto& theme = Menu::GetSingleton()->GetTheme();
+	if (ImGui::BeginTable("##GlobalPerformanceProfiles", 3,
```

Cppcheck (2.21.0)

[error] Returning pointer that will be invalid when returning.

(returnDanglingLifetime)